pyiron/atomistics/job/atomistic.py

```diff
@@ -833,6 +833,14 @@ def energy_tot(self):
     def forces(self):
         return self._job["output/generic/forces"]
 
+    @property
+    def f_max(self):
+        """
+            returns maximum force magnitude of each step which is used for
```

Comment:
No need to write "returns" since this is a property

Reply:
Yes because the user doesn't see this as a function. Maybe also mention that this is less useful for MD calculations.

Reply:
I guess since it's specifically mentioned that it's used for structure optimisation, it's fine, isn't it?